pkg/pfconfig/shared/shared.go

```diff
@@ -277,6 +277,9 @@ var defaultTxLUTConfigs = []TxLUTConfig{
 
 // BuildSX1301Config builds the SX1301 configuration for the given frequency plan.
 func BuildSX1301Config(frequencyPlan *frequencyplans.FrequencyPlan) (*SX1301Config, error) {
+	if len(frequencyPlan.Radios) == 0 {
```

Comment:
This ignores all the other fields that belong here, right? Shouldn't this result in an error instead or at least generate a partially correct config?
As I see it now this is just silently failing and that can't be right.

Reply:
Imo this `BuildSX1301Config` shouldn't be called if there aren't radios. The caller is responsible to check _what_ radios there are before calling functions that are about a specific concentrator.

Reply:
The reason I implemented it this way is because this is now common for all types of config (UDP/LBS/CPF). With the suggested approach, I had to replicate this check thrice. But done.

Reply:
It's good to be explicit